app-hitl/hitl/src/hitl/commands/run_hitl.py

```diff
@@ -5,24 +5,23 @@
 import click
 import geopandas as gpd
 
-from utils.merge_labels import merge_labels_with_task_grid
-from utils.get_hitl_input import get_input
-from utils.notify_intercom import notify
+from ..utils.merge_labels import merge_labels_with_task_grid
+from ..utils.get_hitl_input import get_input
+from ..utils.notify_intercom import notify
 
-from rv.active_learning import active_learning_step
-from rv.io import get_class_config
+from ..rv.active_learning import active_learning_step
+from ..rv.io import get_class_config
```

Comment:
I believe the `..`'s can be replaced by `hitl.` which is less brittle.

Reply:
Makes sense to me. Since we are merging this PR to your branch and we will test all things together in there after the step 3 and 4 implementations are there- I'd say we make the change then and test it again by a `test/` branch on staging.